docs/working-with-projects/working-with-images/sign-images.md

```diff
@@ -63,37 +63,166 @@ Note that Harbor's [garbage collection](../../administration/garbage-collection/
 
 Harbor doesn't support `cosign clean` to remove signatures as Harbor has chosen not to implement tag deletion which is used by `cosign clean`. See the [OCI distribution specification](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#content-management) for more for more information on implantation requirements.
 
-## Use Notation(experimental) to sign artifacts with distribution spec v1.1 mode
-In order to use Notation, generate a certificate for signing artifacts.
+## Use Notation to sign and verify artifacts with distribution spec v1.1 mode
 
+Harbor supports configuring `http` and `https` domain. For security consideration, we use `https` domain name for secure image signing and verification in this section.
```

Comment:
I think we don't need to include Harbor installation steps here because Harbor users should already set it up. Suggest removing line 68 ~ line 117 in this PR.